packages/webcert/src/components/header/WebcertHeaderUnit.tsx

```diff
@@ -1,10 +1,22 @@
-import React from 'react'
-import { AppHeaderUserUnit } from '@frontend/common'
-import { getUser } from '../../store/user/userSelectors'
-import { shallowEqual, useSelector } from 'react-redux'
+import React, { useState } from 'react'
+import { AppHeaderUserUnit, ResourceLinkType } from '@frontend/common'
+import { getTotalDraftsAndUnhandledQuestionsOnOtherUnits, getUser, getUserResourceLinks } from '../../store/user/userSelectors'
+import { shallowEqual, useDispatch, useSelector } from 'react-redux'
 import { User } from '@frontend/common/src'
 import styled from 'styled-components'
 import AlertCircle from '@frontend/common/src/images/AlertCircle'
+import arrow from '@frontend/common/src/images/arrow-down.svg'
+import { updateIsCareProviderModalOpen } from '../../store/user/userActions'
+
+const ArrowDown = styled.img`
```

Comment:
This feels too general to belong in this specific component. Also, was there nothing in the UX framwork that could be used?

Reply:
I did not find anything in the UX framework I could use. Please let me know if you know a better option. But now I have extracted the arrow into its own component.

Reply:
I think you should try to use this one: https://css.inera.se/?path=/story/components-expandables--card. Maybe not all classNames but some. The solution you have done seems to include som styles that are used in the UX-framwork and your component is missing the aria-attributes for an expandable component so it won't be WCAG compliant. If you use the UX-component then you might get this behavior automatically.

Reply:
Since I will have to change color and size of the arrow I don't think it's worth changing the code. Instead I added aria attributes to the ArrowToggle.